src/Flow/ETL/Transformer/RowTransformer.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Flow\ETL\Transformer;
+
+use Flow\ETL\Row;
+use Flow\ETL\Transformer;
+
+/**
+ * @psalm-immutable
+ */
+interface RowTransformer extends Transformer
```

Comment:
I'm looking for a better name for this interface, one that I can think of is `MergeableTransformer`, which shows which transformers can be merged by Bulk Transformation.